### ISSUE TITLE
Scheduler Migrations: "Proxy" commands to schedulers

### ISSUE THREAD

**Issue by kolloch** [mesosphere/marathon#3499]
Expose an sub-endpoint for the scheduler upgrade API which at least allows:

* querying the full deployment plan
* querying the status
* submitting continue, interrupt commands

The proposed end-point is "/v2/apps/<app id>/migration". The only limitation is that because app IDs may contain slashes, it might be difficult to disambiguate this end-point from an app-id ending in "/migration".

The exposed API should resemble the scheduler upgrade API as close as possible to reduce friction while debugging the whole system. It should not just proxy everything though, since we want more control about what commands the user is allowed to send to the scheduler API.

Depends on #3497.

Context: We want to be able to drive scheduler upgrades manually. See [MARATHON-856](https://mesosphere.atlassian.net/browse/MARATHON-856) (private, sorry).

**Comment by kolloch** [comment]
Won't do this.
